src/components/App/index.js

```diff
@@ -1,20 +1,18 @@
 import React, { useState } from 'react';
 import './App.css';
 
-import AlertWrapper from '../AlertWrapper';
-import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
 import Container from 'react-bootstrap/Container';
 import List from '../List';
-import Row from 'react-bootstrap/Row';
 import Toggle from '../Toggle';
 
 function App() {
   const formLink = 'https://docs.google.com/forms/d/1j_Cha4mhNiegs8mDWGcPGQczYlMxnzYiQw8uKAK7ujY/viewform?edit_requested=true#responses';
-  const responseLink = '/#';
+  const responseEmail = 'tktktk';
```

Comment:
Adding a `// TODO` is easy to search for later.

Reply:
Both of these might actually be better suited in a `config.json` and imported here.